### ISSUE TITLE
TopImage

### ISSUE THREAD

**Issue by PrinceBE** [aaronbond/SwipeDeck2#11]
You added insert left and right image!
like this: cardStack.setLeftImage(R.id.left_image);
        cardStack.setRightImage(R.id.right_image);
Can you add setTopImage and setBottom image?

**Comment by aaronbond** [comment]
I assume you want to be able to swipe up and down also? 

currently swipedeck doesn't have that feature but i could add it.

**Comment by PrinceBE** [comment]
yes when i am swiping up or down and image should display can you add this feature ??
